Format.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\format.ps1""" %*"
```

Comment:
❓  Why is this part script of this PR?

Reply:
We are moving to a version of the SDK which contains "dotnet format" as a CLI command. Invoking the dotnet-format local tool is non-intuitive (`dotnet tool run dotnet-format`) and I thought would benefit from these helper scripts.

Reply:
Will move these to a follow up PR.